dashboard/app/controllers/user_levels_controller.rb

```diff
@@ -0,0 +1,30 @@
+class UserLevelsController < ApplicationController
```

Comment:
Or does this correspond directly to models/user_level.rb and am I supposed to look there to know what this class is for?